src/datumaro/plugins/yolo_format/converter.py

```diff
@@ -73,6 +73,26 @@ def rotate_point(x: float, y: float):
     return points
 
 
+def _resolve_subsets(initial_subsets: Dict[str, IExtractor]) -> Dict[str, Iterable]:
+    assert YoloPath.DEFAULT_SUBSET_NAME.lower() != DEFAULT_SUBSET_NAME.lower()
+
+    subsets: Dict[str, Iterable] = {
+        subset_name: subset
+        for subset_name, subset in initial_subsets.items()
+        if subset_name and subset_name != DEFAULT_SUBSET_NAME
+    }
+    for subset_name, subset in initial_subsets.items():
+        if not subset_name or subset_name == DEFAULT_SUBSET_NAME:
+            subset_name = YoloPath.DEFAULT_SUBSET_NAME
+            try:
+                subset_name = next(name for name in subsets if name.lower() == subset_name.lower())
+            except StopIteration:
+                pass
```

Comment:
Why not `if subset_name.lower() in subsets:`?

Reply:
subsets can have names not only with lowercase, e.g. "Train".